packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -7,26 +7,31 @@ import UnitRow from '../UnitRow';
 import UnitRowWithAvatar from '../UnitRowWithAvatar';
 import Security from '../Security';
 import UnitRowSecondaryEmail from '../UnitRowSecondaryEmail';
-import { Account } from '../../models';
 
-export const Settings = ({ account }: { account: Account }) => {
-  const primaryEmail = account.emails.find((email) => email.isPrimary)!;
+import { useAccount } from '../../models';
+
+const {
+  emails,
+  displayName,
+  avatarUrl,
+  passwordCreated,
+  recoveryKey,
+} = useAccount();
+
+export const Settings = () => {
+  const primaryEmail = emails.find((email) => email.isPrimary)!;
```

Comment:
You can use `account.primaryEmail` which comes from here: https://github.com/mozilla/fxa/blob/issues%2F6118/packages/fxa-settings/src/lib/cache.ts#L35